servlet-core/src/main/java/io/micronaut/servlet/http/ServletBodyBinder.java

```diff
@@ -146,8 +149,8 @@ public String getName() {
                                 } else {
                                     final Argument<? extends List<?>> containerType = Argument.listOf(typeArg.getType());
                                     T content = (T) codec.decode(containerType, inputStream);
-                                    final Flowable flowable = Flowable.fromIterable((Iterable) content);
-                                    final T converted = conversionService.convertRequired(flowable, type);
+                                    final Flux flux = Flux.fromIterable((Iterable) content);
```

Comment:
Not sure if it would be better to have `final Publisher pub` instead of `final Flux flux`

Reply:
I changed it to `final Publisher publisher` ... symmetry with the `if` block.